src/Symfony/Bundle/FrameworkBundle/Resources/config/lock.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
+
+        <service id="lock.store.flock" class="Symfony\Component\Lock\Store\FlockStore">
+            <argument type="service">
+                <service class="string">
+                    <factory function="sys_get_temp_dir"/>
```

Comment:
instead of this, what about making the $lockPath arg of FlockStore optional, and make it use `sys_get_temp_dir` internally instead?